docs/source/core_developers_guide/environment_setup.rst

```diff
@@ -144,3 +174,182 @@ install_sphinx
   This plugin installs sphinx, which is required for building the sawtooth
   documentation.
 
+
+Step One: Clone Repository
+==========================
+
+You'll need to have git installed in order to clone the Sawtooth Lake source
+code repository. You can find up-to-date installation instructions here:
+
+* `Git install instructions <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`_
+
+.. note:: 
+
+  When checking out Sawtooth Lake on Windows for use with vagrant, you should
+  take steps to ensure that Windows-style CRLF line endings are not added to
+  the code. The bash scripts used by your vagrant VM will not run correctly 
+  with CRLF line endings. Git uses a configuration setting, *core.autocrlf*,
+  to control whether or not LF-style line endings are automatically converted
+  to CRLF-style line endings. `This setting should be set in such a way that 
```

Comment:
@delventhalz Good points. This "note" is already in our documentation, and went through a lot of iterations already. However, I will try to make it less awkward. For some reason, I was told to not put the exact setting to use, but just reference it (in case it changes?)